monai/utils/jupyter_utils.py

```diff
@@ -147,25 +149,25 @@ def tensor_to_images(name: str, tensor: torch.Tensor):
     each channel separately.
     """
     if tensor.ndim == 3 and tensor.shape[1] > 2 and tensor.shape[2] > 2:
-        return tensor.cpu().data.numpy()
+        return tensor.cpu().data.numpy()  # type: ignore[no-any-return]
     if tensor.ndim == 4 and tensor.shape[2] > 2 and tensor.shape[3] > 2:
         dmid = tensor.shape[1] // 2
-        return tensor[:, dmid].cpu().data.numpy()
+        return tensor[:, dmid].cpu().data.numpy()  # type: ignore[no-any-return]
 
     return None
 
 
 def plot_engine_status(
     engine: Engine,
-    logger,
+    logger: MetricLogger,
```

Comment:
Building the docs doesn't work because of this forward reference of `MetricLogger`. I tried putting it into quotes so `logger: "MetricLogger"` but this is automatically reverted by the pre-commit pyupgrade. I also searched for inline comments to locally disable pyupgrade, but this does not exist.

So my solutions would be one of:
- Completely disable pyupgrade for this file
- (Preferred) Use `Any` for the argument